students/mitek_michal/lesson_06_dictionares_tuples_sets/cubes.py

```diff
@@ -0,0 +1,27 @@
+
+def cubes():
+
+    n_set, m_set = [int(s) for s in input().split()]
+    n_set_colors, m_set_colors = set(), set()
+
+    for i in range(n_set):
+        n_set_colors.add(int(input()))
+
+    for i in range(m_set):
+        m_set_colors.add(int(input()))
+
+    print_set(n_set_colors and m_set_colors)
```

Comment:
Instead of using ```and``` use ```A & B``` or  ```A.intersection(B)```. Now, your code not passing Snakify tests due to this ```and```